lib/jsdom/living/nodes/HTMLAnchorElement-impl.js

```diff
@@ -16,6 +16,22 @@ class HTMLAnchorElementImpl extends HTMLElementImpl {
   set text(v) {
     this.textContent = v;
   }
+
+  get draggable() {
```

Comment:
This shouldn't be here; it's an attribute for HTMLElement, not for HTMLAnchorElement. So only implement it on HTMLElement.